pkg/syslwrapper/app.go

```diff
@@ -121,7 +123,9 @@ func (am *AppMapper) mapAttributes(attributes map[string]*sysl.Attribute) map[st
 func (am *AppMapper) mapTypes(appName string, syslTypes map[string]*sysl.Type) map[string]*Type {
 	simpleTypes := make(map[string]*Type, 15)
 	for typeName := range syslTypes {
-		simpleTypes[typeName] = am.MapType(am.Types[appName+":"+typeName])
+		if am.MapType(am.Types[appName+":"+typeName]) != nil {
+			simpleTypes[typeName] = am.MapType(am.Types[appName+":"+typeName])
```

Comment:
Similar changes to this

Reply:
No worries, will do!

Reply:
Fixed up in https://github.com/anz-bank/sysl/pull/831/commits/49670ff4551670504b4e47b555a6e4b429979239